README.md

```diff
@@ -21,35 +22,44 @@ Then call it
 ```python
 import asyncio
 
-from ahbicht.mapping_results import PackageKeyConditionExpressionMapping
+from ahbicht.mapping_results import PackageKeyConditionExpressionMapping, ConditionKeyConditionTextMapping
 from ahbichtfunctionsclient import HochfrequenzPackageResolver
 from maus.edifact import EdifactFormat, EdifactFormatVersion
 
 
-async def retrieve_package_mwe():
+async def retrieve_package_key_condition_expression_mapping():
     # for a documentation about the purpose of a package resolver, you should read the ahbicht docs
-    package_resolver = HochfrequenzPackageResolver()
+    package_resolver = HochfrequenzPackageResolver(EdifactFormatVersion.FV2204, EdifactFormat.UTILMD)
     # the following data are just hardcoded to provide you a minimal working example
-    package_resolver.edifact_format = EdifactFormat.UTILMD
-    package_resolver.edifact_format_version = EdifactFormatVersion.FV2204
     package_mapping = await package_resolver.get_condition_expression("10P")  # this does an HTTP GET request
     assert isinstance(package_mapping, PackageKeyConditionExpressionMapping)  # the result is ahbicht compatible
 
+package_loop = asyncio.get_event_loop()
+package_loop.run_until_complete(retrieve_package_key_condition_expression_mapping())
 
-loop = asyncio.get_event_loop()
-loop.run_until_complete(retrieve_package_mwe())
+async def retrieve_condition_key_condition_text_mapping():
+    condition_resolver = HochfrequenzPackageResolver(EdifactFormatVersion.FV2204, EdifactFormat.UTILMD)
+    # the following data are just hardcoded to provide you a minimal working example
+    condition_mapping = await condition_resolver.get_condition_expression("56")  # this does an HTTP GET request
+    assert isinstance(condition_mapping, ConditionKeyConditionTextMapping)  # the result is ahbicht compatible
+
+
+condition_loop = asyncio.get_event_loop()
+condition_loop.run_until_complete(retrieve_condition_key_condition_text_mapping())
```

Comment:
die zwei event loops sind nicht falsch, aber auch nicht schön ;)

wie wäre es stattdessen mit sowas?

```python
async def minimal_working_example():
    """ just some demo code """
    await retrieve_condition_key_condition_text_mapping()
    await retrieve_package_key_condition_expression_mapping()

loop = asyncio.get_event_loop()
loop.run_until_complete(minimal_working_example())
```

die event loop ist nur ein hilfskonstrukt um async/await code in main auszuführen. in meinem vorschlag haben wir immer nur noch eine loop, das fände ich schöner